app/plugins.go

```diff
@@ -557,18 +557,14 @@ func (a *App) InitPlugins(pluginPath, webappPath string) {
 
 	utils.RemoveConfigListener(a.PluginConfigListenerId)
 	a.PluginConfigListenerId = utils.AddConfigListener(func(prevCfg, cfg *model.Config) {
```

Comment:
Should we add the config listener in the top of this function before the `!*a.Config().PluginSettings.Enable` check to allow us to enable plugins without restarting the server?

Reply:
The config listener in cmd/platform/server.go should allow for that. I separated them into two because shutting down the plugins cancels this listener. We could probably consolidate them into toss them at the top here and just remove the cancelling of the listener when shutting down plugins